docker/arbitrum/arbitrum.sh

```diff
@@ -40,7 +40,7 @@ down() {
 wait_for_sequencer() {
     local RETRIES=180
     local i=0
-    until docker logs nitro-sequencer-1 2>&1 | grep -q "HTTP server started";
+    until docker logs nitro-sequencer-1 2>&1 | grep "HTTP server started";
```

Comment:
Why remove `-q` here?

Reply:
with q it is not working ok in macos actually it cant find the log with -q option.

Reply:
Yeah, that would also be my preffered course of action.